styles.css

```diff
@@ -0,0 +1,473 @@
+/* stylelint-disable */
```

Comment:
- [ ] Please don't disable the linters, they are here to help you write better code.